src/plots/cartesian/layout_attributes.js

```diff
@@ -721,6 +721,17 @@ module.exports = {
             'outside and vice versa.'
         ].join(' ')
     },
+    ticklabelindex: {
+        valType: 'integer',
+        arrayOk: true,
```

Comment:
One may consider adding `extras: ['all']` here and then display all the minor and major ticks labels when `ticklabelindex` is set to (or include) *all*. 
So please add a comment.
Also if you want to add *minor* option to just show minor tick labels this could become `extras: ['all', 'minor']`.